src/interface/meta.thrift

```diff
@@ -172,6 +172,71 @@ struct DropSpaceReq {
     2: bool if_exists,
 }
 
+enum AdminJobOp {
+    ADD         = 0x01,
+    SHOW_All    = 0x02,
+    SHOW        = 0x03,
+    STOP        = 0x04,
+    BACKUP      = 0x05,
+    RECOVER     = 0x06,
+    INVALID     = 0xFF,
+} (cpp.enum_strict)
+
+struct AdminJobReq {
+    1: AdminJobOp   op
+    2: list<string> paras;
+}
+
+enum JobStatus {
+    QUEUE           = 0x01,
+    RUNNING         = 0x02,
+    FINISHED        = 0x03,
+    FAILED          = 0x04,
+    STOPPED         = 0x05,
+    INVALID         = 0xFF,
+} (cpp.enum_strict)
+
+struct JobDesc {
+    1: i32 id
+    2: string typeAndParas
+    3: JobStatus status
+    4: i64 startTime
+    5: i64 stopTime
+}
+
+struct TaskDesc {
+    1: i32 taskId
+    2: common.HostAddr host
+    3: JobStatus status
+    4: i64 startTime
+    5: i64 stopTime
+    6: i32 jobId
+}
+
+struct BackupJobResult {
+    1: i32 jobNum
+    2: i32 taskNum
+}
+
+struct AdminJobResult {
+    // used in "admin flush" and "admin compact"
```

Comment:
jobId only be used in "flush" and "compact" ?

Reply:
actually this field used when one new job added. 
other job types, which also need a job id in return will use other field. 
for example:  JobDesc::id

Reply:
Update the comments